docs/v0.14/out_s3.txt

```diff
@@ -0,0 +1,196 @@
+# Amazon S3 Output Plugin
+
+The `out_s3` Output plugin writes records into the Amazon S3 cloud object storage service. By default, it creates files on an hourly basis. This means that when you first import records using the plugin, no file is created immediately. The file will be created when the `timekey` condition has been met. To change the output frequency, please modify the `timekey` value in buffer directive. In more detail, please refer to [the time chunk keys in bufferdocument](https://docs.fluentd.org/v0.14/articles/buffer-section#time).
+
+NOTE: This document doesn't describe all parameters. If you want to know full features, check the Further Reading section.
+
+## Installation
+
+`out_s3` is included in td-agent by default. Fluentd gem users will need to install the fluent-plugin-s3 gem using the following command.
+
+    :::term
+    $ fluent-gem install fluent-plugin-s3
+
```

Comment:
How about adding link to [Plugin Management](https://docs.fluentd.org/v0.14/articles/plugin-management)